Assets/MixedRealityToolkit/Utilities/BuildAndDeploy/BuildDeployPreferences.cs

```diff
@@ -18,6 +18,32 @@ public static class BuildDeployPreferences
         private const string EDITOR_PREF_BUILD_DIR = "BuildDeployWindow_BuildDir";
         private const string EDITOR_PREF_INCREMENT_BUILD_VERSION = "BuildDeployWindow_IncrementBuildVersion";
 
+        private const string EDITOR_PREF_LIVE_CUBE_MODEL_LOCATION = "BuildDeployLiveCubeModelLocation";
+
+        private static BuildDeployPreferencesSO _savedPreferences;
+
+        private const string _savedPreferencesLocation = "Assets/SavedBuildDeployPreferences.asset";
+
+        static BuildDeployPreferences()
+        {
+            string[] result = AssetDatabase.FindAssets("SavedBuildDeployPreferences");
```

Comment:
This isn't deterministic. If you use #6315, then it'll write to one file.

If you still want your own ScriptableObject, it should be located in one place. Also look at putting it in the Generated folder